### PR TITLE
Support New Relic Browser metric conditions

### LINKED ISSUE #14: Add support for Browser conditions
I'd like to contribute those!

### PR DESCRIPTION
**Pull request by lukasz-kusnierz**
Should resolve #14

### CODE REVIEW

File: newrelic-alerts-configurator/src/test/java/com/ocadotechnology/newrelic/alertsconfigurator/BrowserConditionConfiguratorTest.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    private static TermsConfiguration.TermsConfigurationBuilder createTermsConfiguration() {
+        return TermsConfiguration.builder()
+            .durationTerm(DurationTerm.DURATION_10)
+            .operatorTerm(OperatorTerm.ABOVE)
+            .priorityTerm(PriorityTerm.WARNING)
+            .timeFunctionTerm(TimeFunctionTerm.ANY)
+            .thresholdTerm(1f);
+    }
+
+    private static BrowserCondition createAppCondition(String conditionName) {
```
Comment (thestyq) [inline newrelic-alerts-configurator/src/test/java/com/ocadotechnology/newrelic/alertsconfigurator/BrowserConditionConfiguratorTest.java]: `createBrowserCondition`

File: newrelic-alerts-configurator/src/main/java/com/ocadotechnology/newrelic/alertsconfigurator/configuration/condition/BrowserCondition.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    public UserDefinedConfiguration getUserDefinedMetric() {
+        return userDefinedConfiguration;
+    }
+
+    public enum Metric {
+        END_USER_APDEX, TOTAL_PAGE_LOAD, PAGE_RENDERING, WEB_APPLICATION, NETWORK, DOM_PROCESSING, REQUEST_QUEUING,
+        AJAX_RESPONSE_TIME, PAGE_VIEWS_WITH_JS_ERRORS, PAGE_VIEW_THROUGHPUT, AJAX_THROUGHPUT, USER_DEFINED
+    }
+
+    public static class BrowserConditionBuilder {
```
[the remark this answers is not in the document]
Comment (lukasz-kusnierz) [reply newrelic-alerts-configurator/src/main/java/com/ocadotechnology/newrelic/alertsconfigurator/configuration/condition/BrowserCondition.java]: :white_check_mark:

File: newrelic-alerts-configurator/README.md [hunk 3]
```diff
@@ -277,6 +279,48 @@ Condition serverMetricCondition = ServerMetricCondition.builder()
     .build();
 ```
 
+#### New Relic Browser metric condition
```
[the remark this answers is not in the document]
Comment (lukasz-kusnierz) [reply newrelic-alerts-configurator/README.md]: :white_check_mark:

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,6 @@
+# 3.1.0 (23.02.2018)
```
[the remark this answers is not in the document]
Comment (lukasz-kusnierz) [reply CHANGELOG.md]: Come on! :wink:

File: newrelic-alerts-configurator/src/test/java/com/ocadotechnology/newrelic/alertsconfigurator/BrowserConditionConfiguratorTest.java (hunk 1 above)
Comment (lukasz-kusnierz) [reply newrelic-alerts-configurator/src/test/java/com/ocadotechnology/newrelic/alertsconfigurator/BrowserConditionConfiguratorTest.java]: :white_check_mark:

**Pull request merged**